app/models/project/phase.rb

```diff
@@ -61,6 +61,10 @@ def visible(user = User.current)
     end
   end
 
+  def any_dates_set?
```

Comment:
I think it should be singular:
```suggestion
  def any_date_set?
```